src/ray/raylet/scheduling_queue.cc

```diff
@@ -6,44 +6,14 @@
 
 namespace {
 
-// Helper function to remove tasks in the given set of task_ids from a
-// queue, and append them to the given vector removed_tasks.
-template <typename TaskQueue>
-void RemoveTasksFromQueue(ray::raylet::TaskState task_state, TaskQueue &queue,
-                          std::unordered_set<ray::TaskID> &task_ids,
-                          std::vector<ray::raylet::Task> *removed_tasks,
-                          std::vector<ray::raylet::TaskState> *task_states = nullptr) {
-  for (auto it = task_ids.begin(); it != task_ids.end();) {
-    if (queue.RemoveTask(*it, removed_tasks)) {
-      it = task_ids.erase(it);
-      if (task_states != nullptr) {
-        task_states->push_back(task_state);
-      }
-    } else {
-      it++;
-    }
-  }
-}
+static const std::array<std::string,
```

Comment:
Hmm I slightly prefer `std::array` here because the number of task queues must be specified, in case we decide to add another `TaskState` in the future.